vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -9,7 +9,7 @@ open OptionsUIHelpers
 
 
 module DefaultTuning = 
-    let SemanticColorizationInitialDelay = 500 (* milliseconds *)
+    let SemanticColorizationInitialDelay = 0 (* milliseconds *)
```

Comment:
Instead of having comments here we could just suffix the value with Msecs? Just a minor style point